packages/amplication-client/src/Application/GithubTileFooter/GithubTileFooter.scss

```
@@ -0,0 +1,37 @@
+@import "../../style/index.scss";
+
+.github-tile-footer {
+  &__user-details {
+    @include flexVerticalCenter();
+    margin-bottom: 12px;
```

please use variables 
```suggestion
    margin-bottom: var(--default-spacing);;
```